pkg/lightning/restore/restore_test.go

```diff
@@ -1550,67 +1550,67 @@ func (s *restoreSchemaSuite) TestRestoreSchemaContextCancel(c *C) {
 }
 
 func (s *tableRestoreSuite) TestCheckClusterIsOnline(c *C) {
-	cases := []struct {
-		mockHttpResponse []byte
-		expectMsg        string
-		expectResult     bool
-		expectWarnCount  int
-		expectErrorCount int
-	}{
-		{
-			[]byte(`{
-				"count": 1,
-				"regions": [
-					{
-						"id": 1,
-						"written_bytes": 11000000
-					}
-				]
-			}`),
-			"(.*)write flow(.*)",
-			false,
-			1,
-			0,
-		},
-		{
-			[]byte(`{
-				"count": 1,
-				"regions": [
-					{
-						"id": 1
-					}
-				]
-			}`),
-			"(.*)Cluster has no other loads(.*)",
-			true,
-			0,
-			0,
-		},
-	}
-
-	ctx := context.Background()
-	for _, ca := range cases {
-		server := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-			_, err := w.Write(ca.mockHttpResponse)
-			c.Assert(err, IsNil)
-		}))
-
-		tls := common.NewTLSFromMockServer(server)
-		template := NewSimpleTemplate()
-
-		url := strings.TrimPrefix(server.URL, "https://")
-		cfg := &config.Config{TiDB: config.DBStore{PdAddr: url}}
-		rc := &Controller{cfg: cfg, tls: tls, checkTemplate: template}
-		err := rc.ClusterIsOnline(ctx)
-		c.Assert(err, IsNil)
-
-		c.Assert(template.FailedCount(Warn), Equals, ca.expectWarnCount)
-		c.Assert(template.FailedCount(Critical), Equals, ca.expectErrorCount)
-		c.Assert(template.Success(), Equals, ca.expectResult)
-		c.Assert(strings.ReplaceAll(template.Output(), "\n", ""), Matches, ca.expectMsg)
-
-		server.Close()
-	}
+	// cases := []struct {
```

Comment:
I prefer to lower case this function name and add a common to minimal the affacted lines

Reply:
directly remove this function call,  When multiple lightning instance work on the cluster, check cluster workload is meaningless